### ISSUE TITLE
Not all enabled duties show their notifications

### ISSUE THREAD

**Issue by Minganie** [MidoriKami/DailyDuty#89]
I have Beast Tribe, Custome Delivery, Doman Enclave and Hunt Marks (Weekly) enabled with Notifications on Login and on Zone Change. However, many notifications are definitely missing. For example, when I teleported just now, it only showed Beast Tribe and Custom Delivery. It's not always the same notifications showing, it's not always the same notification missing. Is there like a cap on the number of notifications? If that is the case, there should be something about that in the readme.

**Comment by MidoriKami** [comment]
I need more information and pictures if possible.

**Comment by Minganie** [comment]
What specific information do you need?
[Image link since Github won't let me past and image](https://imgur.com/a/zz2L52r)

**Comment by MidoriKami** [comment]
Can you show me a picture of your Doman Enclave Status tab, or send your config file

**Comment by MidoriKami** [comment]
Also please attach a copy of your `dalamud.log` Located in `%appdata%\XIVLauncher`

**Comment by Minganie** [comment]
[Doman Enclave tab](https://imgur.com/a/bxBeu8X)
[dalamud.log](https://github.com/MidoriKami/DailyDuty/files/9548967/dalamud.log)

**Comment by MidoriKami** [comment]
Please attach the files located in `%appdata%\XIVLauncher\pluginConfigs\DailyDuty`

**Comment by Minganie** [comment]
[18014498554094098.json.txt](https://github.com/MidoriKami/DailyDuty/files/9551869/18014498554094098.json.txt)
I just renamed it with an added .txt so Github would accept it. No changes to file internals.

**Comment by MidoriKami** [comment]
Ok I think you might be misunderstanding how DailyDuty works.

First off, you have Custom Delivery set to "Mark Complete when Allowances are Equal To 12" meaning, it marks it `Complete` when you have 12, and `Incomplete` when you have anything other than 12.

Notifications are only sent if a module is `Incomplete` and there are tasks available to do.

You can see the relevant information in the `Status` Window, by clicking on the `Status` button or by typing `/dd status`

**Comment by Minganie** [comment]
Then my bug is actually a documentation problem.

I suggest clarifying language in the interface. I can't check the specifics right now since Dalamud is down, but the Custom Delivery page says something like "complete when less than 12" without saying WHAT needs to be less than 12. I could not make sense of the English there no matter how many times I reread it.

I would also suggest modifying the defaults. It should be marked as incomplete as long as you have allowances, not the other way around.

Finally, in the README you refer to the wiki. Except there's nothing in the wiki? Unless I'm looking in the wrong spot? I wouldn't have bothered you if there was some doc for me to read that explains this.

**Comment by MidoriKami** [comment]
The default comparison value was incorrect, thanks for catching that. I have adjusted the default value to make more sense.

![image](https://user-images.githubusercontent.com/9083275/190004179-80d099c8-a5ad-4d10-b7a5-8acf2897dd3d.png)
The image reads as "Mark complete when {Less than} 12 Allowances"

I removed the wiki because I was changing the plugin so much and so often that the information was out of date and incorrect. I have adjusted the information under notifications.
